packages/block-library/src/site-logo/index.php

```diff
@@ -22,8 +22,13 @@ function render_block_core_site_logo( $attributes ) {
 	};
 
 	add_filter( 'wp_get_attachment_image_src', $adjust_width_height_filter );
+
 	$custom_logo = get_custom_logo();
 
+	if ( empty( $custom_logo ) ) {
+		return ''; // Return early if no custom logo is set, avoiding extraneous wrapper div.
+  }
```

Comment:
Sorry Jeff, I broke your code when trying to solve the merge conflict via the GitHub UI 👎 . There is supposed to be a tab here.
```suggestion
	}
```